addons/velero/1.6.2/tmpl-s3-migration-configmap.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: velero-migrate-s3-config
+  labels:
+    app: velero
+data:
+  migrate-s3.sh: |-
+    #!/bin/sh
+    set -euo pipefail
+    export S3_HOST=\$OBJECT_STORE_CLUSTER_IP
```

Comment:
arent these in environment variables set in the kubernetes resource env property using the secret? wont running this through render_yaml_file strip all these vars? also expose the secrets in the config map?

Reply:
this is really the only blocker for me. everything else is more stylistic changes

Reply:
I don't think they get stripped becasue the notation for templates is curly braces, but I agree this doesn't need to be a template. I think I copied from `registry` and it's like that there. Will fix.